test/test_operations.py

```diff
@@ -664,6 +664,21 @@ def test_get_xla_tensor(self):
     self.assertEqual(tx, sx.data.cpu())
 
 
+class TestBinaryCrossEntropyLimitValue(XlaTestCase):
+
+  def test_cross_entropy_loss(self):
+    lossfn = nn.BCELoss()
+    target_cpu = torch.tensor((1.))
```

Comment:
We have an existing harness to test XLA vs CPU:

https://github.com/pytorch/xla/blob/b1e4f93652516e9c497b44e76adf69fe1bc860eb/test/test_operations.py#L1588